synapse/lib/node.py

```diff
@@ -765,14 +770,20 @@ async def pack(self, path=False):
         info = await s_stormtypes.toprim(dict(self.metadata))
         if path:
             info['nodes'] = [node.iden() for node in self.nodes]
+            info['links'] = self.links
+
         return info
 
-    def fork(self, node):
+    def fork(self, node, link=None):
+
+        links = list(self.links)
+        if self.node:
+            links.append((self.node.iden(), link))
```

Comment:
I added it as an optional since I was a bit paranoid about backwards compat. I'm not against making it mandatory, I'm just stepping lightly with any direct API changes.

Reply:
I think we can make it mandatory unless we have a specific reason to allow `null` in the chain :+1: